bootstrap/bpdoc/properties.go

```diff
@@ -250,17 +250,23 @@ func filterPropsByTag(props *[]Property, key, value string, exclude bool) {
 	// len(props) times to this slice will overwrite the original slice contents
 	filtered := (*props)[:0]
 	for _, x := range *props {
-		tag := x.Tag.Get(key)
-		for _, entry := range strings.Split(tag, ",") {
-			if (entry == value) == !exclude {
-				filtered = append(filtered, x)
-			}
+		if hasTag(x.Tag, key, value) == !exclude {
+			filtered = append(filtered, x)
 		}
 	}
 
 	*props = filtered
 }
 
+func hasTag(tag reflect.StructTag, key, value string) bool {
```

Comment:
Can this reuse proptools.HasTag?

Reply:
Not entirely since proptools.HasTag accepts reflect.StructField, and the above call fetches the tag from bpdoc.Property. I replaced the one in bpdoc.go, though.